src/services/members/interfaces/member.ts

```diff
@@ -17,6 +17,12 @@ export interface Member {
   id: UUID;
   name: string;
   email: string;
+}
+
+export interface CompleteMember {
+  id: UUID;
+  name: string;
+  email: string;
```

Comment:
Does it make sense to reuse `Member`? 
```
CompleteMember = Member & { ... }
```

Reply:
@pyphilia You can not do that with interfaces. But I can change them to be type aliases instead.